src/PackageUtilities.js

```diff
@@ -138,6 +139,14 @@ export default class PackageUtilities {
     return packages;
   }
 
+  static filterPackagesThatAreNotUpdated(packagesToFilter, command) {
```

Comment:
Taking the whole `command` here feels a bit gross, but couldn't think of an alternative.